.github/workflows/cross-version-tests.yml

```diff
@@ -3,6 +3,19 @@ name: Cross version tests
 on:
   workflow_dispatch:
     inputs:
+      repository:
+        description: >
+          [Optional] Repository name with owner. For example, mlflow/mlflow.
+           Defaults to the repository that triggered a workflow.
+        required: false
+        default: ""
```

Comment:
An empty string here is intentional. This makes `actions/checkout@v2` default to using the repository that triggered this workflow.

Reply:
If we run this workflow on `mlflow/mlflow`, `repository` defaults to `mlflow/mlflow`. If I run this workflow on my fork, it defaults to `harupy/mlflow`.